crates/router/src/core/admin.rs

```diff
@@ -5,7 +5,7 @@ use common_utils::{
     ext_traits::ValueExt,
 };
 use error_stack::{report, FutureExt, ResultExt};
-use masking::Secret; //PeekInterface
+use masking::{PeekInterface, Secret}; //PeekInterface
```

Comment:
```suggestion
use masking::{PeekInterface, Secret};
```

Remove this comment.